src/main/java/CompanyList.java

```diff
@@ -5,4 +5,16 @@
 public class CompanyList {
 
     ArrayList<Company> companies = new ArrayList<>();
+
+    public void list(){
+        if (companies.size() == 0){
```

Comment:
Perhaps include a space between the closing parenthesis and the opening curly brace.